src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  cursor: pointer;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect on the card is a bit too much. Scaling the card to 1.2 times its original size may cause the layout to break or look strange. Also, changing the color of the text on hover is not a good practice as it may decrease readability.